Jenkinsfile

```diff
@@ -25,39 +25,58 @@ pipeline {
                 }
             }
         }
-        stage('Build artifacts') {
+        stage('Sonar') {
             steps {
                 script {
-                    if (env.BRANCH_NAME ==~ /(master|develop)/ || env.TAG_NAME) {
-                        DOCKER_TAGS = ['master': 'latest', 'develop': 'develop']
-                        withCredentials([usernamePassword(credentialsId: 'nexus-soramitsu-rw', usernameVariable: 'login', passwordVariable: 'password')]) {
-                          env.DOCKER_REGISTRY_URL = "https://nexus.iroha.tech:19004"
-                          env.DOCKER_REGISTRY_USERNAME = "${login}"
-                          env.DOCKER_REGISTRY_PASSWORD = "${password}"
-                          env.TAG = env.TAG_NAME ? env.TAG_NAME : DOCKER_TAGS[env.BRANCH_NAME]
-                          sh "./gradlew dockerPush"
+                    if (env.BRANCH_NAME == 'develop') {
+                        withCredentials([string(credentialsId: 'SONAR_TOKEN', variable: 'SONAR_TOKEN')]){
+                            sh(script: "./gradlew sonarqube -x test --configure-on-demand \
+                                -Dsonar.links.ci=${BUILD_URL} \
+                                -Dsonar.github.pullRequest=${env.CHANGE_ID} \
+                                -Dsonar.github.disableInlineComments=true \
+                                -Dsonar.host.url=https://sonar.soramitsu.co.jp \
+                                -Dsonar.login=${SONAR_TOKEN} \
+                                ")
                         }
                     }
                 }
             }
         }
-        stage('Sonar') {
-                  steps {
-                    script {
-                      if (env.BRANCH_NAME == 'develop') {
-                        withCredentials([string(credentialsId: 'SONAR_TOKEN', variable: 'SONAR_TOKEN')]){
-                          sh(script: "./gradlew sonarqube -x test --configure-on-demand \
-                            -Dsonar.links.ci=${BUILD_URL} \
-                            -Dsonar.github.pullRequest=${env.CHANGE_ID} \
-                            -Dsonar.github.disableInlineComments=true \
-                            -Dsonar.host.url=https://sonar.soramitsu.co.jp \
-                            -Dsonar.login=${SONAR_TOKEN} \
-                            ")
-                          }
-                      }
+        stage('Push artifacts') {
+            steps {
+                script {
+                    if (env.BRANCH_NAME ==~ /(master|develop)/) {
```

Comment:
```suggestion
                    if (env.BRANCH_NAME in DOCKER_TAGS ) {
```

Reply:
It isn't fixed

Reply:
I missed this line, fixed from sonar stage one

Reply:
I don’t understand. You marked it as resolved. You reject the suggested change or accept?